### ISSUE TITLE
[UPnP Menu] Maintaining the old genre

### ISSUE THREAD

**Issue by tesshucom** [tesshucom/jpsonic#2663]
Prerequisites: #2662

### Overview

Modify the previous Genre (Subsonic style). This feature will not be actively improved in the future. It has not yet been decided whether it will be removed.

### Details

Only minimal modifications.

#### Fix 1

Delete the count display. Reason·· No longer needed.

 - Because the counts for legacy servers are incorrect. Jpsonic has already been improved to enable accurate counting.
 - The reason this was implemented was partly for testing purposes and partly for debugging purposes. (We can see that the count doesn't work properly)
 - This is a menu item that will not be actively maintained, so we want to keep the implementation code to a minimum.

![image](https://github.com/user-attachments/assets/2c6d7322-00ac-4c2b-9b8a-4ae40c5b8740)

#### Fix 2

Improve genre to album expansion. Currently, for genres that contain some special strings, accurate view cannot be performed. (There is no problem with the newly implemented Menu.)
